### PR TITLE
Bug 1051991 - Exclude batch tasks from front page

### PR DESCRIPTION
**Pull request by tessie**

### CODE REVIEW

**Comment by tessie** [comment]
@bobsilverberg.ready for review

File: oneanddone/base/views.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
 class HomeView(TaskMustBeAvailableMixin, FilterView):
     template_name = 'base/home.html'
-    queryset = Task.objects.filter(difficulty=Task.BEGINNER)
+    queryset = Task.objects.filter(difficulty=Task.BEGINNER,batch=None)
```
Comment (bobsilverberg) [inline oneanddone/base/views.py]: Missing a space between `Task.BEGINNER,` and `batch=None`.

**Comment by bobsilverberg** [comment]
Nice work @tessie. This one was so close to being merged without any comments from me, but you left out a space! :)

We should talk about getting a git commit hook in place for you that will check your code for problems like this and prompt you to fix them before you can even commit the code.

**Comment by tessie** [comment]
@bobsilverberg .I must have used flake8 .ready for next review

**Comment by bobsilverberg** [comment]
Looks good, thanks @tessie.

**Pull request merged**